lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -492,6 +492,52 @@ static bool isOkToBitSelectFrom(Value v) {
   return false;
 }
 
+/// Construct a access path which points at the element given by `fieldID`.
+/// TODO: This function should be removed once HW/SV dialects implement
+/// fieldID utilities like FIRRTL dialect.
```

Comment:
Can you open an issue to track this? I agree that this should be short lived while support for fieldID is added to HW/SV.

Reply:
Opened https://github.com/llvm/circt/issues/2419